libs/staking/src/lib/hooks/use-liquid-staking-actions.ts

```diff
@@ -134,10 +120,6 @@ export function useLiquidStakingUndelegate() {
         return txResponse;
       } catch (error) {
         console.error('Undelegation failed:', error);
-        posthog?.capture('undelegate_failed', {
```

Comment:
Why removed?

Reply:
because event already sent from here

https://github.com/haqq-network/frontend/blob/9a0c6b1961e1f9c2b21dbb4d5ce18fc68f5e1a47/libs/staking/src/lib/components/stride/liquid-staking-undelegate-modal-hooked.tsx#L135-L141